vertx-mysql-client/src/main/java/io/vertx/mysqlclient/MySQLChangeUserOptions.java

```diff
@@ -0,0 +1,249 @@
+package io.vertx.mysqlclient;
+
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.json.JsonObject;
+import io.vertx.mysqlclient.impl.MySQLCollation;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.vertx.mysqlclient.MySQLConnectOptions.*;
+
+/**
+ * Authentication options for MySQL CHANGE_USER command.
+ */
+@DataObject(generateConverter = true)
+public class MySQLChangeUserOptions {
```

Comment:
I think we should call it `MySQLAuthOptions`

Reply:
updated in https://github.com/eclipse-vertx/vertx-sql-client/pull/420/commits/0d847058c7205b566256252cb111b79f2a61b698.